### PR TITLE
add blog post about feedback for error messages

### PR DESCRIPTION
**Pull request by bishabosha**
screencapture-localhost-4000-blog-2023-10-17-feedback-wanted-error-messages-html-2023-10-17-15_20_58

### CODE REVIEW

File: blog/_posts/2023-10-17-feedback-wanted-error-messages.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jamie Thompson (Scala Center)
+title: "We want to know which errors bug you the most"
```
Comment (adpi2) [inline blog/_posts/2023-10-17-feedback-wanted-error-messages.md:5]: Maybe a shorter title

```suggestion
title: "Which errors bug you the most"
```

File: blog/_posts/2023-10-17-feedback-wanted-error-messages.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+description: "Feedback is wanted on confusing or unhelpful error messages"
+---
+
+## We want your feedback to improve error reporting
```
Comment (adpi2) [inline blog/_posts/2023-10-17-feedback-wanted-error-messages.md:9]:
```suggestion
## Your feedback to improve error reporting
```

File: blog/_posts/2023-10-17-feedback-wanted-error-messages.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+![example error message for a non-exhaustive pattern in a for-comprehension]({{ site.baseurl }}/resources/img/blog/2023-10-error-messages/splash-image.jpg)
+**TL;DR: [click here][issue-template] to tell the Scala 3 core maintainers which error messages are unhelpful, confusing or otherwise suboptimal.**
+
+### Why are we asking you
+
+From [suggesting to import missing extension methods](https://docs.scala-lang.org/scala3/book/ca-extension-methods.html#import-extension-method) and more, Scala 3 tries to improve the ways that warnings and errors are reported to the user.
+The idea being that when the user engages with reported problems, it should feel like having a conversation with a helpful tutor. By helping users to get back on track, we can make Scala easier to learn and make Scala friendlier to newcomers, and also keep experienced users more productive.
```
Comment (adpi2) [inline blog/_posts/2023-10-17-feedback-wanted-error-messages.md:17]:
```suggestion
The idea being that when the user engages with reported problems, it should feel like having a conversation with a helpful tutor. By helping users to get back on track, we can make Scala easier to learn and friendlier to newcomers, and also keep experienced users more productive.
```

File: blog/_posts/2023-10-17-feedback-wanted-error-messages.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+We know there is much more that can be done to continually improve the situation. As well as working to [expose diagnostics programatically](https://github.com/lampepfl/dotty/issues/14904), we are starting a new campaign to seek out confusing, annoying or unhelpful error/warning messages.
+
+As compiler authors, we do try to provide good messages, but its possible for there to be shortcomings.
```
Comment (adpi2) [inline blog/_posts/2023-10-17-feedback-wanted-error-messages.md:21]:
```suggestion
As compiler authors, we do try to provide good messages, but it is possible for there to be shortcomings.
```

Comment (bishabosha) [reply blog/_posts/2023-10-17-feedback-wanted-error-messages.md]: thanks!

File: blog/_posts/2023-10-17-feedback-wanted-error-messages.md (hunk 3 above)
Comment (bishabosha) [reply blog/_posts/2023-10-17-feedback-wanted-error-messages.md]: thanks!

File: blog/_posts/2023-10-17-feedback-wanted-error-messages.md (hunk 2 above)
Comment (bishabosha) [reply blog/_posts/2023-10-17-feedback-wanted-error-messages.md]: I made it shorter

**Review by SethTisue — APPROVED** [review]
👍

**Pull request merged**